src/components/LoginRegister/LoginInput.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { useApolloClient } from '@apollo/client';
+import { useDispatch } from 'react-redux';
+import { Input, Link, Button, Heading, Flex, VStack, Box } from '@chakra-ui/react';
+import { login } from '../../data/actions/auth';
+
+const LoginInput = () => {
+  const client = useApolloClient();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const dispatch = useDispatch();
+
+  const handleClick = async e => {
+    e.preventDefault();
+    dispatch(login(email, password, client));
```

Comment:
not sure how much work this is but would be great if we can add in simple error message if the login fails (if it's too much refactoring we can do this in another PR)

i.e. if the login response is `null` show a generic error message like "Oops! Check your credentials and try again."

Reply:
Added a simple error message for both login and register!